### ISSUE TITLE
Translation to spanish

### ISSUE THREAD

**Issue by profesorfalken** [RunGp/Nvidiux#7]
Hi RunGp, 

I was thinking in translating your software to spanish language. I have installed qt dev tools and I have started translating a qm file.
However when I try to test I find that the language is always in french, even when my system is configured in US english: http://www.cippu.org/pic/syUeFSSdSDiCrMiY

Maybe I am doing something wrong?

Anyway, keep up with you good work. You tools works perfect with my MSI TF 750ti!

**Comment by RunGp** [comment]
Hello
For change language go to nvidiux settings or modify ~/.nvidiux/conf.xml  change line lang (fr_FR to en_EN)
For add spanish translation traduce all english sentence of this file (nvidiux_sp_Sp.ts)
[ts_spainfile.zip](https://github.com/RunGp/Nvidiux/files/139604/ts_spainfile.zip)
And send me the result file.
Thanks for help.

**Comment by profesorfalken** [comment]
Ok, I will try to have it this week (I have renamed the file to 'es_ES').

In my case I tried to execute directly from code I was shocked to find it in french (même si je parle la langue). 

I see that the config is only loaded if the file exist: 
    if os.path.isfile(self.home + "/.nvidiux/conf.xml"):
        self.loadNvidiuxConf()

So the settings are not charged. 
Shouldn't you load in that case some default values? (locale.getDefaultLocale() for the language, for example). In fact you set the default the lang to "en_EN" but it does not work even work in my case because the nvidiuxTranslator is not still loaded.

In fact, I think that is enough if you create the translator before instancing the NvidiuxApp object (L1448 - nvidiux.py).

In summary, I would propose: 
-Set default lang to locale.getDefaultLocale()
-Load the nvidiuxTranslator before instancing NVidiuxApp (swap the calls).

Its is just an idea :-) 

Have a good day

**Comment by RunGp** [comment]
Yes bad practice
I developped my "small" program in "franglais" (code in english all message in french) ( I am french) normaly it's for me or just for french forum.
But with time nvdiux became an useful program and after having discussed on the french ubuntu forums I have decide share "my little program" for all.
My project for nvidiux 2.0 retranslated all french text in english and add comments

Thanks for advice test it tomorrow

Cote français tu le parle couramment ?

**Comment by RunGp** [comment]
relaunch

**Comment by profesorfalken** [comment]
Hello, 
Here you are the file translated into spanish as promised.
[spanish_file](https://github.com/profesorfalken/profesorfalken.github.io/blob/master/files/nvidiux_es_ES.ts)

Cote français je dirais que je me débrouille pas trop mal. J'habite dans le sud de la France depuis trois ans.

**Comment by RunGp** [comment]
Ok thanks for this translation file

Super tu est donc trilangue :) pour d’éventuel futur version tu accepterais de continuer la traduction en espagnol ?

**Comment by profesorfalken** [comment]
Oui bien sur, tu peux compter sur moi :-)

**Comment by RunGp** [comment]
Bonjour 
il faudrait que tu me donne une adresse de contact pour que je puise t'envoyer les fichiers à traduire.
Il ne reste que cet étape pour sortir la 1.3
[langfile.zip](https://github.com/RunGp/Nvidiux/files/172167/langfile.zip)

Merci

**Comment by profesorfalken** [comment]
Ok pas de souci. 
Tu peux m'envoyer les fichiers a cette adresse liée a mon profil: profesorfalken.github@gmail.com et on reste en contacte.
Apparemment il n'y a pas trop des choses a traduire. Je pense que je pourrai le faire même aujourd'hui.
